interview-questions.md

```diff
@@ -8,62 +8,85 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+Your answer: 
 
-  Researched answer:
+A parameter is the value that is defined in a functional block and is basically used whenever we need to pass a value to a particular functional block.
+
+The arguments are the actual values that you send your function to use when you invoke it.
 
+Researched answer: 
```

Comment:
Parameters and values have such a close relationship. They blend and merge in my head often.